src/Microsoft.AspNetCore.Razor.LanguageServer/RemoteProjectItem.cs

```diff
@@ -21,6 +21,8 @@ public RemoteProjectItem(string filePath, string physicalPath)
 
         public override string PhysicalPath { get; }
 
+        public override string RelativePhysicalPath => FilePath;
```

Comment:
Is `FilePath` here relative? 👀 Is that how it is in other `RazorProjectItem`s in Razor?

Reply:
FilePath is relative, but after re-looking at the code the RelativePhysicalPath does have one differentiation (it ensures there's no leading `/`). I'll play around with that in Razor.VSCode as well.